packages/perseus/src/components/tex-buttons.jsx

```diff
@@ -252,24 +252,22 @@ const buttonSets: ButtonSets = {
     ],
 };
 
+export type ButtonSetsType = $ReadOnlyArray<$Keys<typeof buttonSets>>;
+
 type TexButtonProps = {|
-    sets: $ReadOnlyArray<$Keys<typeof buttonSets>>,
+    sets: ButtonSetsType,
     onInsert: (Inserter) => void,
     className?: string,
     convertDotToTimes?: boolean,
 |};
 
-const buttonSetsType = PropTypes.arrayOf(PropTypes.oneOf(_(buttonSets).keys()));
+const buttonSetsPropType = PropTypes.arrayOf(
+    PropTypes.oneOf(_(buttonSets).keys()),
+);
```

Comment:
Given this is an old-school `PropTypes` value, can we just get rid of it?  seems like the field on line 270 is never used.

Reply:
It's used in `expression-editor`: `buttonSets: TexButtons.buttonSetsType`

I'll see how hard it'll be to Flow type that.

Reply:
So TexButtons is passing PropTypes in a way I don't think we can pass Flow Types (by attaching them to the class). Since `expression-editor` is in a different packages, I think I'd need to change the type exports of the `perseus` package. I'm thinking this might be beyond the scope of my current work (unless you have a quick fix).

https://khanacademy.atlassian.net/browse/LC-626

Reply:
Ok.  I'd say **don't** go down the rabbit hole now! :)